packages/rawlink/rawlink.0.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+homepage: "https://github.com/haesbaert/rawlink"
+bug-reports: "https://github.com/haesbaert/rawlink/issues"
+license: "ISC"
+dev-repo: "https://github.com/haesbaert/rawlink.git"
+build: ["sh" "build.sh"]
+depends: [
+  "ocamlfind"
```

Comment:
arghh, thanks, I pushed the {build} tag down by mistake. Should I cancel this pull request and make a new one ?